searchobject/src/main/java/org/jalau/at18/searchobject/model/facedetector/FaceDetect.java

```diff
@@ -0,0 +1,138 @@
+package org.jalau.at18.searchobject.model.facedetector;
+/**
+ * Copyright (c) 2022 Jala University.
+ *
+ * This software is the confidential and proprieraty information of Jalasoft
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * Licence agreement you entered into with Jalasoft
+ */
+
```

Comment:
do we need this white space? For separating one thing from another, just leave 1 space.

Reply:
already erased